src/components/formSections/FormTerms.js

```diff
@@ -85,10 +100,14 @@ class FormTerms extends React.Component {
           name="customProperties"
           component={TermsListField}
           availableTerms={this.state.terms}
+          onError={this.handleError}
         />
       </Accordion>
     );
   }
 }
 
-export default injectIntl(FormTerms);
+export default compose(
+  injectIntl,
+  stripesConnect
```

Comment:
It seems like `withStripes` would be enough here, no? You're not using stripes-connect.